packages/myst-config/src/site/types.ts

```diff
@@ -43,7 +50,7 @@ type ManifestProjectItem = {
   banner?: string | null;
   bannerOptimized?: string;
   tags?: string[];
-  exports?: Export[];
+  exports?: SiteExport[];
 };
```

Comment:
I get a type error in the theme when reconciling downloads 👇

![image](https://github.com/executablebooks/mystmd/assets/1473646/67ecb37d-214d-4cec-ac26-8378b58c803b)

downloads are also expected on pages? or is this reconcilation in the theme wrong? @fwkoch

Reply:
Those `ManifestProjectItems` never actually have either exports or downloads. I removed `exports` from the type and deleted that line from `myst-theme`